src/Tasks/Microsoft.NET.Build.Tasks/LockFileExtensions.cs

```diff
@@ -112,6 +112,19 @@ public static LockFileTargetLibrary GetLibrary(this LockFileTarget lockFileTarge
             return exclusionList;
         }
 
+        public static HashSet<PackageIdentity> GetTransitiveExclusionList(
```

Comment:
I don't understand why this is called `ExclusionList`.  It seems like you are just returning all the transitive dependencies of the specified package.